pyblish_nuke/plugins/select_host.py

```diff
@@ -1,12 +0,0 @@
-import pyblish.api
-
-
-@pyblish.api.log
-class SelectHostVersion(pyblish.api.Selector):
```

Comment:
Think it might have been legacy from before hosts were part of the api. Not using it, so its safe to remove for me.